app/api/events.py

```diff
@@ -33,16 +35,36 @@ class Meta:
         self_view_many = 'v1.event_list'
         inflect = dasherize
 
+    @validates_schema
+    def validate_date(self, data):
+        if data['starts_at'] >= data['ends_at']:
+            raise UnprocessableEntity({'pointer': 'ends_at'}, "ends_at should be after starts_at")
```

Comment:
```
{
    "errors": [
        {
            "status": 422,
            "source": {
                "pointer": "ends_at"
            },
            "detail": "ends_at should be after starts_at",
            "title": "Unprocessable Entity"
        }
    ],
    "jsonapi": {
        "version": "1.0"
    }
}
```
About pointer and parameter, we've been using pointer, parameter and source inside `source` in the error message interchangeably andwe should decide on one of them, what do you think?